packages/core/src/core/plugins/moduleCacheControl.ts

```diff
@@ -0,0 +1,69 @@
+import type { RsbuildPlugin, Rspack } from '@rsbuild/core';
+
+class CacheControlRspackPlugin {
+  apply(compiler: Rspack.Compiler) {
+    const { RuntimeModule } = compiler.webpack;
+    class RetestCacheControlModule extends RuntimeModule {
+      constructor() {
+        super('rstest_cache_control');
+      }
+
+      override generate() {
+        return `
+global.setupIds = [];
+
+function __clean_rstest_core_cache__() {
+  if (typeof __webpack_require__ === 'undefined') {
+    return;
+  }
+  delete __webpack_module_cache__['@rstest/core'];
+
+  global.setupIds.forEach((id) => {
+    delete __webpack_module_cache__[id];
+  });
+}
+
+global.__clean_rstest_core_cache__ = __clean_rstest_core_cache__;
```

Comment:
Can we access `__webpack_require__` instead of using `global` to call `__clean_rstest_core_cache__`, which means we could call `__webpack_require__.__clean_rstest_core_cache__` directly. And also, we'd better prefix all Rstset runtime method with `rstest_` (`rstest_clean_core_cache`)